content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -106,13 +106,10 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. This two proxies on the data path add about 6.3 ms to the 90th percentile latency at 1000 requests per second.
-The server-side proxy alone adds 1.7 ms to the 90th percentile latency.
-
 ### Latency for Istio {{< istio_release_name >}}
 
-The default configuration of Istio {{< istio_release_name >}} adds 6.3 ms to the 90th percentile latency of the data plane over the baseline.
+Inside the mesh, a request traverses the client-side proxy and then the server-side
+proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2), the two proxies add about 2.8 ms and 2.7 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
```

Comment:
at the top of the page we say "The Envoy proxy adds 6.3 ms to the 90th percentile latency." this seems to contradict this?

Reply:
yeah.... let me fit it, actually, to some extent, I think we need to reorganize this page